examples/simpler_transaction.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+
+use Illuminate\Database\DatabaseManager;
+use SaatchiArt\BentoBoxDDD\Adapters\Repositories\ArtworkRepositoryMysqlImpl;
+use SaatchiArt\BentoBoxDDD\Adapters\Repositories\Transactions\TransactionFactory;
+use SaatchiArt\BentoBoxDDD\Adapters\Repositories\UserRepositoryMysqlImpl;
+
+
+$databaseManager = new DatabaseManager();
+$artworkRepository = new ArtworkRepositoryMysqlImpl($databaseManager);
+$userRepository = new UserRepositoryMysqlImpl($databaseManager);
+
+$transactionFactory = new TransactionFactory($databaseManager);
+
+
+
+$simpleTransaction = $transactionFactory->makeTransaction('central1');
+
+$resultIfAny = $simpleTransaction->executeSimpleTransaction(static function () use ($artworkRepository, $userRepository) {
```

Comment:
No protection. If one of the repositories does NOT support transactions OR share the same connection, then you risk having inconsistent data if a problem occurs.